.eslintrc

```diff
@@ -22,6 +22,6 @@
   },
 
   "rules": {
-    "comma-dangle": ["always-multiline"]
+    "comma-dangle": [2, "always-multiline"]
```

Comment:
@wilsonpage seems there was an error with comma-dangle rule, the option was missing (0 - off, 1 - warning, 2 - error). What should be the value to set? setting it to 2, we have more than 100 lint errors :S